edb/server/args.py

```diff
@@ -150,6 +151,7 @@ def __str__(self):
     ServerConnTransport.TCP: ServerAuthMethod.Scram,
     ServerConnTransport.TCP_PG: ServerAuthMethod.Scram,
     ServerConnTransport.HTTP: ServerAuthMethod.JWT,
+    ServerConnTransport.SIMPLE_HTTP: ServerAuthMethod.Scram,
```

Comment:
This is a bit misleading, as we're not sending any "Salted Challenges" as in SCRAM. Should we have a new `ServerAuthMethod.Password` enum?

Reply:
Yeah, let me test if the config patches that would be necessary to backport it to 4.x work.
(Since it's actually an object in the config system)

Reply:
Yeah I think this should work

Reply:
... maybe. I'm getting some really weird errors testing the new auth extension after in an place upgrade... We'll see.

Reply:
Ok got it